media/media.css

```diff
@@ -1,43 +1,108 @@
-.cards {
+* {
+	margin: 0;
+	box-sizing: border-box;
```

Comment:
Can we work a bit on code formatting here?

I see sometimes 1 space, sometimes 2, 3 or 4. Maybe unify to 2 spaces?